models/src/main/protobuf/DeployServiceV1.proto

```diff
@@ -133,6 +136,19 @@ message ListeningNameDataPayload {
   int32 length                         = 2;
 }
 
+// listenForDataAtPar
+message ListeningParDataResponse {
```

Comment:
We can call it like in WebAPI `RhoDataResponse` and correspondingly the payload also.